setup.py

```diff
@@ -28,7 +28,7 @@
 ]
 
 setup(
-    name='dag',
+    name='py-ipld-dag',
```

Comment:
Don't you think that saying `py` in a Python package's name is a little bit redundant? AFAIR other IPFS/IPLD packages ommit the language prefix

Reply:
Actually not, just following the convention that the other implementations are also following.

Reply:
Fair enough